pkg/services/ngalert/notifier/config_test.go

```diff
@@ -0,0 +1,122 @@
+package notifier
+
+import (
+	"errors"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/grafana/alerting-api/pkg/api"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_PersistTemplates(t *testing.T) {
```

Comment:
Isn't underscore in test names normally used to indicate a certain test case, as opposed to what's being tested? That's the convention I'm familiar with at least.